SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -214,6 +216,29 @@ void SaveSchemaDatabase()
 	});
 }
 
+void LoadDefaultGameMode()
+{
+	// Get the default GameMode from the DefaultEngine.ini
+	FString DefaultGameModePath;
```

Comment:
Did you try using `UObject* GameModeObject = LoadObject<UObject>(nullptr, *UGameMapsSettings::GetGlobalDefaultGameMode());`?

Reply:
It seems like there are other GameModes/Classes we can load from there. Lets just load all of them.

Reply:
There is also the server one which I've added. The other classes get loaded with the GameMode.